tiledb/sm/misc/constants.cc

```diff
@@ -49,7 +49,7 @@
 #include "tiledb/sm/enums/compressor.h"
 #include "tiledb/sm/enums/datatype.h"
 #include "tiledb/sm/enums/serialization_type.h"
-#include "tiledb/sm/misc/utils.h"
+#include "tiledb/sm/misc/rectangle.h"
```

Comment:
Is this header necessary?--given that there's no adjustment of namespaces in the rest of the file.

Reply:
Looks like an oversight. I'll remove.